### PR TITLE
update to cookies content

### PR DESCRIPTION
**Pull request by caitlin-reidy**
### Trello card
https://trello.com/b/lRXllOHK/git-website-sprint-board

### Context

Our cookies were recently updated to reflect changes needed with adopting VWO and MS Clarity, however Gemma identified there may be some other aspects needing review.

### CODE REVIEW

**Comment by caitlin-reidy** [comment]
Need to review whether or not 'cookies' should be capitalised or not, both used at different places in the page but i'm not sure what the right version is? @GayeCord

**Comment by gemmadallmandfe** [comment]
> Thanks Caitlin, the changed to the cookie preference page look good to me. However (and this could be me), the modal page still looks like it has the incorrect info within it and doesn't look to have returned to the 'accept cookies' or 'set cookie preferences' view image

@caitlin-reidy I think you've updated the right page now - but the changes to the modal (the acceptance component) are still on this PR - they just need removing!

**Comment by caitlin-reidy** [comment]
Have deleted Internet explorer and added safari now @GayeCord

**Comment by GayeCord** [comment]
Thanks for the changes to the web browser help section @caitlin-reidy. I've just found a couple of additional errors in the Tracking and Performance section, please can you look at these too:
- MUID cookie - the description is missing the leading I
- SM cookie - has the wrong description
- SRM_B - has the wrong description 
- FPID - showing as **S**FPID. The S need to be removed.
Thanks

**Comment by caitlin-reidy** [comment]
@GayeCord have corrected those mistakes now

File: app/views/pages/cookies.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Cookies Policy
 ---
 
-Last updated: 12 July 2023
+Last updated: 1 December 2023
```
Comment (gemmadallmandfe) [inline app/views/pages/cookies.md:5]:
```suggestion
Last updated: 7 December 2023
```

File: app/views/pages/cookies.md [hunk 2]
```diff
@@ -28,79 +28,78 @@ For the purposes of this Cookies Policy:
...
 
-- 'You' means the individual accessing or using the Website, or a company, or any legal entity on behalf of which such individual is accessing or using the Website, as applicable
+- 'You' means the individual accessing or using the website, or a company, or any legal entity on behalf of which such individual is accessing or using the website, as applicable.
 
-## The use of the cookies
+## The use of the Cookies
 
-The information below shows the Cookies that are set by [education.gov.uk](https://www.gov.uk/government/organisations/department-for-education) and the third-party services we use. You may refuse the use of Cookies by selecting the appropriate settings on your browser. However, please note that if you do this you may not be able to use the full functionality of this Website.
+The information below shows the Cookies that are set by [education.gov.uk](https://www.gov.uk/government/organisations/department-for-education) and the third-party services we use. You may refuse the use of Cookies by selecting the appropriate settings on your browser. However, please note that if you do this you may not be able to use the full functionality of this website.
```
Comment (gemmadallmandfe) [inline app/views/pages/cookies.md:35]:
```suggestion
The information below shows the Cookies that are set by [education.gov.uk](https://www.gov.uk/government/organisations/department-for-education) and the third-party services we use. You may refuse the use of Cookies by selecting the appropriate settings on your browser. However, please note that if you do this you may not be able to use the full functionality of this Website.
```

File: app/views/pages/cookies.md [hunk 3]
```diff
@@ -28,79 +28,78 @@ For the purposes of this Cookies Policy:
 
 - 'Website' refers to Get Into Teaching, accessible from https://getintoteaching.education.gov.uk/
 
-- 'You' means the individual accessing or using the Website, or a company, or any legal entity on behalf of which such individual is accessing or using the Website, as applicable
+- 'You' means the individual accessing or using the website, or a company, or any legal entity on behalf of which such individual is accessing or using the website, as applicable.
```
Comment (gemmadallmandfe) [inline app/views/pages/cookies.md:31]:
```suggestion
- 'You' means the individual accessing or using the Website, or a company, or any legal entity on behalf of which such individual is accessing or using the Website, as applicable.
```

**Review by martyn-w — APPROVED** [review]
👍  LGTM

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me - have tested the Return to England changes as well that were merged to this branch instead of production (they are ready to go into prod as well)

**Review by GayeCord — APPROVED** [review]

**Pull request closed without merging**